tests/login.test.js

```diff
@@ -0,0 +1,146 @@
+const chai = require('chai');
+const Reports = require('../src/models/reports');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {server} = require("../src/app");
```